### DISCUSSION TITLE
Nodered

### DISCUSSION THREAD

**Discussion by crazynightgriffin** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#471] [General]
Can anyone share their flows for sending temp changes and turning the pump on and off from node-red within home assistant.  I tried directly to the item and also MQTT to no avail.

Cheers

**Comment by visualapproach** [comment]
Just curious: Why do you want to use node red for that? I know I did that too at first because YAML was new to me, but now you can do it easily in the web UI of HA. Automation Trigger - click and select layzspa temp. Action - click service turn on/off layzspa heat.

**Reply by crazynightgriffin** [reply]
I am looking to set routines say 0:30 - 4:30 set to 40'c all other times set to 38'c  but then with the option to change the setting on the fly say when we away without having to delete the queue and redo all the routines.

**Reply by visualapproach** [reply]
The easiest way if you have home assistant would be to use ha instead of the esp queue. Create an automation that change the temperature as you want. Then you can add an entity card to a dashboard with that automation. When leaving on holiday you flip the switch to stop the automation and can set things manually.
